conf/evolutions/reversions/033-tasktype-recommendedConfiguration.sql

```diff
@@ -1,7 +1,7 @@
 START TRANSACTION;
 
 DROP VIEW webknossos.taskTypes_;
-ALTER TABLE webknossos.taskTypes DROP COLUMN recommendedConfiguration JSONB;
+ALTER TABLE webknossos.taskTypes DROP COLUMN recommendedConfiguration;
```

Comment:
Is this simply a fix, missing in another PR?

Reply:
Exactly, I assume nobody tested the reversion before…